### DISCUSSION TITLE
PAT Basic 1052. 卖个萌 (20) (C语言实现) | PAT Solution in C

### DISCUSSION THREAD

**Discussion by xlucn** [xlucn/PAT#70] [General]
https://oliverlew.github.io/PAT/Basic/1052.html 

 题目

**Comment by Huuuuu123** [comment]
为什么在dev c++编译器上，你的代码第一种 1 1 2 2 2 会出现“Are。。”，是我编译器编码问题吗，我调试了很久没看明白，有些字符的值就是0，在PAT上是没有问题。

**Comment by xlucn** [comment]
> @Huuuuu123
> 为什么在dev c++编译器上，你的代码第一种 1 1 2 2 2 会出现“Are。。”，是我编译器编码问题吗，我调试了很久没看明白，有些字符的值就是0，在PAT上是没有问题。

请以OJ编译环境设置为参考，不同编译器对C标准的实现程度不同，标准外的更无法保证
